avaframe/tests/test_tmp1Ex.py

```diff
@@ -11,4 +11,5 @@ def test_tmp1ExMain(capfd):
 
     tmp1Ex.tmp1ExMain(cfg)
     out, err = capfd.readouterr()
+    print(out)
```

Comment:
This is a debug statement, right?

Reply:
oups forgot that... is it possible to get the out from a log.info or do we have to use print() (see the tmp1ExMain)